src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/RefreshLearner/RefreshLearnerCommandHandler.cs

```diff
@@ -11,28 +10,37 @@ namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.RefreshLea
 {
     public class RefreshLearnerCommandHandler : ICommandHandler<RefreshLearnerCommand>
     {
+        private readonly ILogger<RefreshLearnerCommandHandler> _logger;
         private readonly IApprenticeshipIncentiveDomainRepository _incentiveDomainRepository;
         private readonly ILearnerService _learnerService;
         private readonly ILearnerDomainRepository _learnerDomainRepository;
 
         public RefreshLearnerCommandHandler(
+            ILogger<RefreshLearnerCommandHandler> logger,
             IApprenticeshipIncentiveDomainRepository incentiveDomainRepository,
             ILearnerService learnerService,
             ILearnerDomainRepository learnerDomainRepository)
         {
+            _logger = logger;
             _incentiveDomainRepository = incentiveDomainRepository;
             _learnerService = learnerService;
             _learnerDomainRepository = learnerDomainRepository;
         }
 
         public async Task Handle(RefreshLearnerCommand command, CancellationToken cancellationToken = default)
         {
-            Domain.ApprenticeshipIncentives.ApprenticeshipIncentive incentive = await _incentiveDomainRepository.Find(command.ApprenticeshipIncentiveId);
+            var incentive = await _incentiveDomainRepository.Find(command.ApprenticeshipIncentiveId);
+            var learner = await _learnerDomainRepository.GetOrCreate(incentive);
 
-            Learner learner = await _learnerDomainRepository.GetOrCreate(incentive);
+            _logger.LogInformation("Start Learner data refresh from Learner match service for ApprenticeshipIncentiveId: {ApprenticeshipIncentiveId}, ApprenticeshipId: {ApprenticeshipId}, UKPRN: {UKPRN}, ULN: {ULN}",
+                learner.ApprenticeshipIncentiveId, learner.ApprenticeshipId, learner.Ukprn, learner.UniqueLearnerNumber);
 
             SubmissionData submissionData = null;
             var learnerData = await _learnerService.Get(learner);
+
+            _logger.LogInformation("End Learner data refresh from Learner match service for ApprenticeshipIncentiveId: {ApprenticeshipIncentiveId}, ApprenticeshipId: {ApprenticeshipId}, UKPRN: {UKPRN}, ULN: {ULN}",
```

Comment:
This isn't needed to support the requirement.

Reply:
Additional logging added as per request from test team and personal experience while supporting and debugging issues in TEST